functions/script.js

```diff
@@ -0,0 +1,31 @@
+console.log(plus(1, 1));
+console.log(plus(1, "1"));
+console.log(plus(1, +"1"));
+console.log(plus(1, false));
+console.log(plus(1, true));
+console.log(div(10, 3));
+console.log(plus());//undefined->NaN
+console.log(plus(1));//undefined->NaN
+console.log(plus(null, null));//null->0
+console.log(plus("test ", "me"));
+console.log(plus("test ", "me", "again"));
+
+function plus(a, b) {
+  return a + b;
+}
+
+function minus(a, b) {
+  return a - b;
+}
+
+function multiply(a, b) {
+  return a * b;
+}
+
+function divide(a, b) {
+  return a / b;
+}
+
+function div(a, b) {
+  return (a - a % b) / b;
```

Comment:
`%` sometimes returns result you don't want
please try `3.5 % 2`.
if I will pass float number in your function in can give me wrong result, please test it